app/uk/gov/hmrc/gform/ApplicationLoader.scala

```diff
@@ -71,6 +76,10 @@ class ApplicationModule(context: Context)
   protected val configModule = new ConfigModule(context, playBuiltInsModule, wsClient)
   protected val auditingModule = new AuditingModule(configModule, akkaModule, applicationLifecycle)
 
+  addAppMap(
```

Comment:
Why a Map? can we not inject HmrcTrackingConsentSnippet through an argument?

Reply:
Remove code.